contrib/bump-version.sh

```diff
@@ -13,8 +13,8 @@ fi
 CURRENT_VERSION=${CURRENT_VERSION:-"v1.4.0_coreos.0"}
 TARGET_VERSION=${1}
 
-CURRENT_VERSION_BASE=${CURRENT_VERSION:0:`expr index $CURRENT_VERSION _` -1}
-TARGET_VERSION_BASE=${TARGET_VERSION:0:`expr index $TARGET_VERSION _` -1}
+CURRENT_VERSION_BASE=${CURRENT_VERSION%%_*}
+TARGET_VERSION_BASE=${TARGET_VERSION%%_*}
```

Comment:
This deletes everything from the first underscore using bash variable substitutions.